docs/docs/pods.md

```diff
@@ -0,0 +1,580 @@
+
+## Pod Definitions
+Pods are configured via a JSON pod definition, which is similar to an [application definition]({{ site.baseurl }}/docs/application-basics.html). You must declare the resources required by each container in the pod because Mesos, not Marathon, determines how and when to perform isolation for all resources requested by a pod. 
+See the [Examples](#examples) section for complete pod definitions.
+
+### Executor Resources
+
+The executor runs on each node to manage the pods. By default, the executor reserves 32 MB and .1 CPUs per pod for overhead. Take this overhead into account when declaring resource needs for the containers in your pod. You can modify the executor resources in the `executorResources` parameter of your pod definition.
+
+```json
+{
+    "executorResources": {
+    "cpus": 0.1,
+    "mem": 64,
+    "disk": 10mb
+    }
+}
+```
+
+### Secrets
+
+Specify a secret in the `secrets` field of your pod definition. The argument should be the fully qualified path to the secret in the store.
+
+```json
+{
+    "secrets": {
+        "someSecretName": { "source": "/fully/qualified/path" }
+    }
+}
+```
+
+If you are not using Marathon on DC/OS, you will also need to enable the `secrets` feature via Marathon command-line flags and have a secrets plugin implementation.
+
+### Volumes
+
+Pods support ephemeral volumes, which are defined at the pod level. Your pod definition must include a `volumes` field that specifies at least the name of the volume and a `volumeMounts` field that specifies at least the name and mount path of the volume.
+
+```json
+{
+	"volumes": [
+		{
+			"name": "etc"
+		}
+	]
+}
+```
+
+```json
+{
+	"volumeMounts": [
+		{
+			"name": "env",
+			"mountPath": "/mnt/etc"
+		}
+	]
+}
+```
+
+Pods also support host volumes. A pod volume parameter can declare a `host` field that references a pre-existing file or directory on the agent.
+```json
+{
+	"volumes": [
+		{
+			"name": "local",
+			"host": "/user/local"
+		}
+	]
+}
+```
+
+### Containerizers
+
+Marathon pods support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/). The Mesos containerizer [supports multiple images, such as Docker](http://mesos.apache.org/documentation/latest/container-image/). [Learn more about running Docker containers on Marathon]({{ site.baseurl }}/docs/native-docker.html).
+
+The following JSON specifies a Docker image for the pod:
+
+```json
+{  
+   "image":{  
+      "id":"mesosphere/marathon:latest",
+      "kind":"DOCKER",
+      "forcePull":false
+   }
+}
+```
+
+### Network
+
+# Create and Manage Pods
+
+Use the `/v2/pods/` endpoint to create and manage your pods. [See the full API spec]({{ site.baseurl }}/docs/generated/api.html#v2_pods).
+
+## Create
+
+```bash
+ $ curl -X POST -H "Content-type: application/json" -d@/dev/stdin http://<ip>:<port>/v2/pods <mypod>.json
```

Comment:
should be `curl -X POST -H "Content-type: application/json" -d@<mypod>.json http://<ip>:<port>/v2/pods`